data/json/recipes/recipe_food.json

```
@@ -6639,6 +6661,29 @@
     "tools": [ [ [ "surface_heat", 10, "LIST" ] ] ],
     "components": [ [ [ "ketchup", 2 ] ], [ [ "seasoning_mild", 2, "LIST" ] ], [ [ "sugar_standard", 1, "LIST" ] ], [ [ "vinegar", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "honey_mustard",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": "17 m",
+    "charges": 3,
+    "batch_time_factors": [ 40, 5 ],
+    "book_learn": [ [ "cookbook", 1 ], [ "mag_cooking", 1 ], [ "cookbook_foodfashions", 1 ] ],
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
+    "proficiencies": [ { "proficiency": "prof_food_prep" } ],
+    "components": [
+      [ [ "any_butter_or_oil", 2, "LIST" ] ],
+      [ [ "honey", 1 ] ],
```

```suggestion
      [ [ "honey", 1 ] ], 
```
Honey isn't an item.  There are a bunch of different types of honey in game like "honey_bottled", but honey by itself isn't an item.

---

huh... wasn't aware of that, thanks for the heads up. What do people usually do for a replacement for honey in these situations?

---

Is there a “honey” list? In requirements? If not let me know and I’ll try to make one on vacation.

---

Found it! Going with

> [ [ "honey_bottled", 1 ], [ "honeycomb", 1 ], [ "honey_glassed", 1 ] ]

I think it will have the same effect.